static/js/util/util.js

```diff
@@ -256,6 +258,21 @@ export function getLocation(profile: Profile): string {
   return `${subCountryLocation}, ${countryLocation}`;
 }
 
+/**
+ * returns the user's most recent (or current) employer
+*/
+export function getEmployer(profile: Profile): Maybe<string> {
+  let entries = workEntriesByDate(profile.work_history);
+  if ( _.isEmpty(entries) ) {
+    return Nothing();
+  }
+  let [, entry] = entries[0];
+  if ( entry.company_name ) {
+    return Just(entry.company_name);
+  }
+  return Nothing();
+}
+
```

Comment:
I don't have an opinion about the `util` folder. I know you feel strongly about it, so perhaps you should open a separate issue to discuss renaming it / moving things around.